drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -378,7 +388,7 @@ def can_see_client_details?(user)
         },
       )
       Project.import!([:report_id, :project_id], involved_projects.map { |p_id| [id, p_id] }, batch_size: 5_000)
-      ClientProject.import!(project_clients.to_a.compact, batch_size: 5_000)
+      ClientProject.import!(project_clients.to_a.compact.map { |attr| ClientProject.new(attr) }, batch_size: 5_000)
```

Comment:
This ensures that the shape of the project_clients hashes are consistent (by converting them to ClientProject records.  I was having issues with saving since sometimes it included household_type and sometimes not.  I'll add a comment.